app/assets/javascripts/discourse/app/controllers/user-topics-list.js

```diff
@@ -36,10 +29,7 @@ export default Controller.extend(BulkTopicSelection, {
     this.set("application.showFooter", !this.get("model.canLoadMore"));
   },
 
-  @discourseComputed("incomingCount")
-  hasIncoming(incomingCount) {
-    return incomingCount > 0;
-  },
+  incomingCount: alias("pmTopicTrackingState.newIncoming.length"),
```

Comment:
not a very big fan of alias here. It means you can do `set("incomingCount", xxx)` and it would mutate `pmTopicTrackingState.newIncoming.length` this is risky behavior and should be used very rarely.

I usually prefer to use reads in this case: https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/reads

Reply:
O so many computed helpers in Ember now xD. I've updated the PR.